tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -2,6 +2,14 @@
 
 from coalib.bearlib.aspects.collections import aspectlist
 from coalib.bearlib.aspects.Metadata import Metadata
+from coalib.bears.LocalBear import LocalBear
```

Comment:
Maybe we do need it there but I don't think we should have any dependencies on other test files. We could have it in both places